app/models/cart_validations.rb

```diff
@@ -105,36 +105,27 @@ def check_cookie_limit
     errors
   end
 
-  def check_max_items(count_hash, relevant, count_method)
```

Comment:
Is there a reason why we're no longer running this validation? Or is that something you're planning on adding back?